packages/massa-proto-cli/src/as-gen.ts

```diff
@@ -0,0 +1,135 @@
+import { writeFileSync } from 'fs';
+import { spawnSync } from 'child_process';
+import path from 'path';
+import { ProtoFile } from './protobuf';
+
+/**
+ * Converts the given protobuf type to the related assembly script type.
+ *
+ * @param type - the protobuf type
+ *
+ * @returns the assembly script type
+ */
+function convertTypeToAS(type: string): string {
+  switch (type) {
+    case 'bool':
+      return 'bool';
+    case 'int32':
+      return 'i32';
+    case 'int64':
+      return 'i64';
+    case 'uint32':
+      return 'u32';
+    case 'uint64':
+      return 'u64';
+    case 'float':
+      return 'f32';
+    case 'double':
+      return 'f64';
+    case 'string':
+      return 'string';
+    default:
+      throw new Error(`Unsupported type: ${type}`);
+  }
+}
+
+/**
+ * Creates imports for the caller file generated by {@link generateAsCall}
+ *
+ * @param protoData - the proto file data
+ *
+ * @returns the multiline import statements as a string
+ */
+function generateAsImports(protoData: ProtoFile): string {
+  let imports = `import { encode${protoData.funcName}Helper, ${protoData.funcName}Helper } `;
+  imports += `from './${protoData.funcName}Helper';\n`;
+
+  if (protoData.resType !== null) {
+    imports += `import { decode${protoData.funcName}RHelper, ${protoData.funcName}RHelper }`;
+    imports += `from './${protoData.funcName}RHelper';\n`;
+  }
+
+  imports += 'import { call } from "@massalabs/massa-as-sdk";\n\n';
+
+  return imports;
```

Comment:
improves readibility by:
- using template litteral to give an overview of the produced code
- remove optional new lines. they will be added on the caller side
```suggestion
  let responseTypeImports = "";
  
  if (protoData.resType !== null) {
    responseTypeImports = `import { decode${protoData.funcName}RHelper, ${protoData.funcName}RHelper };
    from './${protoData.funcName}RHelper';`;
  }

  return `import { encode${protoData.funcName}Helper, ${protoData.funcName}Helper };
  from './${protoData.funcName}Helper';
  ${responseTypeImports}

  import { call } from "@massalabs/massa-as-sdk";`;
```